IPython/core/alias.py

```diff
@@ -169,6 +169,10 @@ def __call__(self, rest=''):
         if cmd.find('%l') >= 0:
             cmd = cmd.replace('%l', rest)
             rest = ''
+         
+        if cmd.find('%%s') >= 1:
+            cmd = cmd.replace('%%s', '%s')
```

Comment:
I think you need to move this check inside the `if nargs==0:` block immediately below, so that it only happens if we're not formatting arguments into the command.